purchase_exception/wizard/purchase_exception_confirm.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Copyright 2020 Camptocamp SA
+# Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PurchaseExceptionConfirm(models.TransientModel):
+    _name = "purchase.exception.confirm"
+    _inherit = ["exception.rule.confirm"]
```

Comment:
`_description` is required to pass runbot.

Reply:
just added